### PR TITLE
Docs v3

### PR DESCRIPTION
**Pull request by williebsweet**
Restructure the docs

### CODE REVIEW

File: docs/support.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Support
+---
+## Datafold Open-Source
+
+- GitHub [Issues](https://github.com/datafold/data-diff/issues) and [Discussions](https://github.com/datafold/data-diff/discussions)
+
+## Datafold Cloud
+
+- Email support
```
Comment (leoebfolsom) [inline docs/support.md:10]: - Should each of these (or at least Email and Intercom) under Datafold Cloud have links?

**Comment by leoebfolsom** [comment]
The "Jump To" section has the same options as the top menu. 

- It's an extra mental step to realize that.
- Until you realize that, it's overwhelming.
- Once you've realized it, you can completely ignore "Jump To." 

I think we're better off without it. However, it's not worth debating if others strongly disagree.

Screenshot 2023-04-03 at 15 12 35

**Review by leoebfolsom — APPROVED** [review]
I think this is good as is.

I left a few notes for changes that I think are low-hanging fruit BUT they could also be fast follow-ons.

I think the new structure is a major improvement, and we should merge it to let it go live and have a positive impact right away, and then make additional refinements over time.

(I have some wordsmithing in mind that definitely should not hold up the major incremental improvement that shipping this would deliver.)

**Comment by leoebfolsom** [comment]
These changes will have to be incorporated (Slim CI -> Slim Diff and mentioning Hightouch models) https://github.com/datafold/datafold-docs/pull/111

File: docs/support.md (hunk 1 above)
Comment (williebsweet) [reply docs/support.md:10]: No - Intercom is in the app, so there is no link and I don't want our support email out there for anyone.

File: docs/development_testing.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+<!-- ***Replace*** with a quick primer on the importance of Development Testing - what problem is being solved and how our solution is 10x better. -->
+When developing models, it's hard to understand how changes to your code impact the data it produces. Development Testing with Data Diff can bring more visibility to that process.
+
+## Getting Started
+
+### Open-source
+
+<!-- ***Replace*** with a quick primer on how open-source solves the problem above. -->
+Use open source data diff to get a summary of how the model you are working on compares to production - items like columns added/removed and the number of rows added/removed/updated.
```
[the remark this answers is not in the document]
Comment (williebsweet) [reply docs/development_testing.md]: Fixed here: https://github.com/datafold/datafold-docs/pull/143/commits/b3955037cfdf90a292b29637632b716fb2c8a606

File: docs/development_testing/cloud.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+### Generate an API Key
+
+To generate a personal API key, navigate to **Settings** &rarr; **Account** and click **Create API Key**. 
+
+Copy and export your API Key as an environment variable. We suggest storing it in a file like `.zshrc` or `.bash_profile`, but you can also run the command below directly in your project.
+
+  ```bash
+  export DATAFOLD_API_KEY=XXXXXXXXX
+  ``` 
+
```
[the remark this answers is not in the document]
Comment (williebsweet) [reply docs/development_testing/cloud.md:46]: fixed here: https://github.com/datafold/datafold-docs/pull/143/commits/b8589c787a2d8e1288812cb3f07087d9381ada47

**Review by kylemcnair — APPROVED** [review]
Looks good. 

It's difficult to give this amount of change meaningful review, but I explored the deploy preview and didn't find anything negative of note. Merging as is and iterating as needed in separate PRs seems a good path forward (echoing Leo).

**Comment by leoebfolsom** [comment]
@kylemcnair to your point, and just to clarify, my review was "this is objectively better than the current state, so unless there are reasons not to, let's merge it" 🙌

**Pull request merged**